app/controllers/posts_controller.rb

```diff
@@ -3,7 +3,7 @@ class PostsController < ApplicationController
 
   # GET /posts or /posts.json
   def index
-    @posts = Post.all
+    @posts = Post.where.not(static_id: :about)
```

Comment:
I think we should create a default scope for the common posts. (posts without staic id)
This would enable us to use Post.all and if we create a new static_id we only need to update it in one place.

More information in the documnetation:
https://guides.rubyonrails.org/active_record_querying.html#applying-a-default-scope

Reply:
Gonna do it!